source/compression.asm

```diff
@@ -99,9 +99,7 @@ ENDM
 
 Decompress::
     M_DecompressionLoadValues
-    M_DecompressionMainBody Decompress_HandleChunk_CopyRawBytes, \
-                            Decompress_HandleChunk_Rle, \
-                            Decompress_HandleChunk_Reference
+    M_DecompressionMainBody Decompress_HandleChunk_CopyRawBytes, Decompress_HandleChunk_Rle, Decompress_HandleChunk_Reference
```

Comment:
@obskyr I just saw your comment here: https://github.com/obskyr/hamtaro-in-pieces/issues/1#issuecomment-379503783

Should I revert this? :sweat_smile: